.changes/v3.13.0/1273-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Fix `vcd_catalog_media` resource so it doesn't hang on successfully completed tasks [GH-1273]
```

Comment:
Please amend comment to contain the details. I mean that now we're adding wait for "success" as an alternative to wait for "100%".

Reply:
Done, let me know how it looks now, if I should change it.